cli/http_util.rs

```diff
@@ -120,8 +145,33 @@ pub fn fetch_string_once(
       .get(CONTENT_TYPE)
       .map(|content_type| content_type.to_str().unwrap().to_owned());
 
-    let body = response.text().await?;
-    return Ok(FetchOnceResult::Code(body, content_type));
+    let etag = response
+      .headers()
+      .get(ETAG)
+      .map(|etag| etag.to_str().unwrap().to_owned());
```

Comment:
Is empty string returned if `etag` header is not present?

Reply:
If not provided it will be `Option::None`